.gitignore

```diff
@@ -1,2 +1,5 @@
 target
 Cargo.lock
+
+# JetBrains IDE files (e.g. RustRover)
+.idea
```

Comment:
This seems like something that is more appropriate for your home directory's `.gitignore` than every project you contribute to.

Reply:
🤦 TIL that you can put a `.gitignore` in your home directory. I'll remove this, thanks.

I also noticed that `Vec`'s impl of `std::io::Write` can be updated to use this method; I'll push another commit with that change tomorrow.